pkg/cluster/builder.go

```diff
@@ -225,3 +226,27 @@ func buildEKSARelease(release *releasev1.EksARelease, bundle *releasev1.Bundles)
 	}
 	return eksaRelease
 }
+
+// GetBundlesManifest builds a Bundles from a FileSpecBuilder.
+func GetBundlesManifest(b FileSpecBuilder) (*releasev1.Bundles, error) {
+	mReader := b.createManifestReader()
+
+	bundlesManifest, err := b.getBundles(mReader)
+	if err != nil {
+		return nil, errors.Wrapf(err, "getting Bundles manifest")
+	}
+
+	return bundlesManifest, nil
+}
+
+// BuildEKSARelease builds an EKSARelease from a FileSpecBuilder and bundles.
+func BuildEKSARelease(b FileSpecBuilder, bundles *releasev1.Bundles) (*releasev1.EKSARelease, error) {
```

Comment:
why does "build release" take a bundles?
Each release it already tied to a bundles by definition